classification.yaml

```diff
@@ -0,0 +1,791 @@
+  status: feature-incomplete
+  priority: 1a
+  notes:
+    Needs to actually have an Alert Production compute environment, and butler support for, but it's a given that once we have one this requirement will be satisfied.
+DMS-MWBT-REQ-0068:
+  status: feature-complete
+  priority: 1a
+  notes:
+    Demonstrated by regular reprocessing runs at NCSA and DP0.2 production.
+DMS-MWBT-REQ-0070:
+  status: feature-complete
+  priority: 1a
+  notes:
+    Demonstrated by any Science Platform notebook that uses the butler.
+DMS-MWBT-REQ-007:
+  status: feature-complete
+  priority: 1a
+  notes:
+    Demonstrated by regular reprocessing runs at NCSA.
+DMS-MWBT-REQ-0093:
+  status: feature-incomplete
+  priority: 2
+  missing:
+    Needs provenance work, to be described on DMTN-205.
+DMS-MWBT-REQ-0094:
+  status: feature-incomplete
+  priority: 1a
+  missing:
+    Needs provenance work, to be described on DMTN-205.  Not clear how much this is still considered a Data Backbone Responsibility, though.
+DMS-MWBT-REQ-0095:
+  status: feature-incomplete
+  priority: 1a
+  missing:
+    Needs provenance work, to be described on DMTN-205.
+DMS-MWBT-REQ-0096:
+  status: feature-incomplete
+  priority: 2
+  missing:
+    Needs provenance work, to be described on DMTN-205.
+DMS-MWST-REQ-0001:
+  status: feature-complete
+  notes:
+    This is a fundamental part of the design of PipelineTask.
+DMS-MWST-REQ-0002:
+  status: feature-complete
+  notes:
+    This is a fundamental part of the design of PipelineTask.
+DMS-MWST-REQ-0018:
+  status: feature-complete
+  notes:
+    Satisfied by BPS plugin system; we have plugins for many workflow systems already.
+DMS-MWST-REQ-0019:
+  status: feature-incomplete
+  missing:
+    I'm not sure all of the listed environments are needed (or distinct; e.g. do we need anything special for CI?), but we at least need a near-real-time environment for AP to satisfy this.
+DMS-MWST-REQ-0020:
+  status: feature-complete
+  notes:
+    Requirement is an attempt to require a QuantumGraph generation algorithm that is common to all execution environments, without actually saying QuantumGraph generation.
+DMS-MWST-REQ-0021:
+  status: feature-complete
+  notes:
+    Satisfied by existence of QuantumGraph generation code.
+DMS-MWST-REQ-0022:
+  status: feature-complete
+  notes:
+    Satisfied by QuantumGraph being serializable.
+DMS-MWST-REQ-0023:
+  status: feature-complete
+  notes:
+    This is just how the execution system works.
+DMS-MWST-REQ-0025:
+  status: feature-complete
+  notes:
+    We do logging in pipetask.
+DMS-MWST-REQ-0024:
+  status: feature-incomplete
+  missing:
+    Needs provenance work, to be described on DMTN-205.
+DMS-MWST-REQ-0026:
+  status: unclear
+  notes:
+    I'm not even sure what this means; it references some "originally proposed provenance mechanism" that _isn't_ what the requirement tries to capture.
+    It _might_ mean "actual" vs "predicted" linkage, which is something we do intend to provide, however.
```

Comment:
At least needs clarification on what it means, if the authors remember the intent better than I do.